SPECS-EXTENDED/rubygem-sys-filesystem/rubygem-sys-filesystem.spec

```diff
@@ -0,0 +1,73 @@
+%global gem_name sys-filesystem
+
+Name:           rubygem-%{gem_name}
+Version:        1.4.3
+Release:        6%{?dist}
+Summary:        Interface for gathering filesystem information
+
+License:        Apache-2.0
+Vendor:         Microsoft Corporation
+Distribution:   Azure Linux
+URL:            https://rubygems.org/gems/sys-filesystem
+Source:         https://github.com/djberg96/sys-filesystem/archive/refs/tags/sys-filesystem-1.4.3.tar.gz#/rubygem-sys-filesystem-1.4.3.tar.gz
```

Comment:
Information purpose:

Verified that `github.com/djberg96` is official org of `rubygems-sys-filesytem` package. Ref: https://rubygems.org/gems/sys-filesystem/versions/1.5.3 (Owners information)